plugins/modules/ec2_vol_info.py

```diff
@@ -51,9 +51,66 @@
 
 '''
 
-# TODO: Disabled the RETURN as it was breaking docs building. Someone needs to
-# fix this
-RETURN = '''# '''
+RETURN = '''
+volumes:
+    description: Volumes that match the provided filters. Each element consists of a dict with all the information related to that volume.
+    type: list
+    elements: dict
+    returned: always
+    contains:
+        attachment_set:
+            description: Information about the volume attachments.
+            type: dict
+            sample: {
+                "attach_time": "2015-10-23T00:22:29.000Z",
+                "deleteOnTermination": "false",
+                "device": "/dev/sdf",
+                "instance_id": "i-8356263c",
+                "status": "attached"
+            }
+        create_time:
+            description: The time stamp when volume creation was initiated.
+            type: str
+            sample: "2015-10-21T14:36:08.870Z"
+        encrypted:
+            description: Indicates whether the volume is encrypted.
+            type: bool
+            sample: False
+        id:
+            description: The ID of the volume.
+            type: str
+            sample: "vol-35b333d9"
+        iops:
+            description: The number of I/O operations per second (IOPS) that the volume supports.
+            type: int
+            sample: null
+        size:
+            description: The size of the volume, in GiBs.
+            type: int
+            sample: 1
+        snapshot_id:
+            description: The snapshot from which the volume was created, if applicable.
+            type: str
+            sample: ""
+        status:
+            description: The volume state.
+            type: str
+            sample: "in-use"
+        tags:
+            description: Any tags assigned to the volume.
+            type: dict
+            sample: {
+                env: "dev"
+                }
+        type:
+            description: The volume type. This can be gp2, io1, st1, sc1, or standard.
+            type: str
+            sample: "standard"
+        zone:
+            description:
```

Comment:
```suggestion
            description: The Availability Zone of the volume
```

It feels wrong to leave the description here blank when we do know what it is.